server/aws-lsp-partiql/src/server/hover-info/hoverDictionary.ts

```diff
@@ -0,0 +1,96 @@
+export const hoverDictionary = {
+    keyword: {
+        as: [
+            '### AS',
```

Comment:
For these short hover snippets, not sure we need a header. The user already knows which token they're hovering on so it might be a bit cluttering to also have the token name in the tooltip? 🤔

Reply:
Change to a better looking as follows:
<img src="https://github.com/user-attachments/assets/edf2a39e-4775-4c3c-b121-be7184b69ea0" width="450">